tutorialb/Rytan.pl

```diff
@@ -88,13 +88,13 @@ sub REVIEW {
-    quest::popup("Combat Abilities", "<c \"#FFFF00\">Using Combat Abilities:</c><br>For a description of what your combat abilities can do, <c \"#00FFFF\">left click</c> on the S button in your <c \"#00FF00\">Combat Abilities Window</c>, then select the name of the combat ability in the window that pops up. The description will appear in the lower portion of this <c \"#00FF00\">Combat Skills Window</c>.<br><br>To use an ability, select your target and then <c \"#00FFFF\">left click</c> on the hotkey you have created in your <c \"#00FF00\">Combat Abilities Window</c> or <c \"#00FF00\">Hotbar</c>.<br><br><c \"#FFFF00\">Endurance:</c><br>Most combat abilities require endurance to cast or to keep active. Endurance is represented by the yellow bar below your hit point meter in the <c \"#00FF00\">Player Window</c>. Much like hit points, endurance will regenerate over time, but unlike hit points, they will regenerate at the same rate whether you are sitting or standing as long as you are not actively attacking something.<br><br>If you are in a rested state and sitting (there will be a yellow swirl in your <c \"#00FF00\">Player Window</c> when this is true), both endurance and hit points regenerate at a much faster rate than normal.<br><br>If you wish to read these lessons again, simly ask Rytan to [review] them.<br><br><c \"#F07F00\">Click 'OK' to continue</c>");
+    quest::popup("Combat Abilities", "<c \"#FFFF00\">Using Combat Abilities:</c><br>For a description of what your combat abilities can do, <c \"#00FFFF\">left click</c> on the S button in your <c \"#00FF00\">Combat Abilities Window</c>, then select the name of the combat ability in the window that pops up. The description will appear in the lower portion of this <c \"#00FF00\">Combat Skills Window</c>.<br><br>To use an ability, select your target and then <c \"#00FFFF\">left click</c> on the hotkey you have created in your <c \"#00FF00\">Combat Abilities Window</c> or <c \"#00FF00\">Hotbar</c>.<br><br><c \"#FFFF00\">Endurance:</c><br>Most combat abilities require endurance to cast or to keep active. Endurance is represented by the yellow bar below your hit point meter in the <c \"#00FF00\">Player Window</c>. Much like hit points, endurance will regenerate over time, but unlike hit points, they will regenerate at the same rate whether you are sitting or standing as long as you are not actively attacking something.<br><br>If you are in a rested state and sitting (there will be a yellow swirl in your <c \"#00FF00\">Player Window</c> when this is true), both endurance and hit points regenerate at a much faster rate than normal.<br><br>If you wish to read these lessons again, simly ask Rytan to [" . quest::saylink("review") . "] them.<br><br><c \"#F07F00\">Click 'OK' to continue</c>");
```

Comment:
IIRC, saylinks don't work in popups

Reply:
yep - got that already when doing another test. Thought I could correct that faster.